docs/griptape-cloud/structures/index.md

```diff
@@ -0,0 +1,44 @@
+# Structures
+
+Use [Structures](https://cloud.griptape.ai/structures) to host any python code in the cloud in seconds.
+
+## Create a Structure
+
+1. First [Connect Your GitHub Account](https://cloud.griptape.ai/account)
+1. Install the [Griptape Cloud GitHub app](https://github.com/apps/griptape-cloud/installations/new/) to any account or organization you'd like to pull code in from
```

Comment:
Should be `2.`

Reply:
Not in markdown. You just specify them with numbers. This will render with 1. and 2.

Reply:
TIL!